src/sfizz/MidiState.cpp

```diff
@@ -113,7 +119,7 @@ float sfz::MidiState::getLastVelocity() const noexcept
 
 void sfz::MidiState::insertEventInVector(EventVector& events, int delay, float value)
 {
-    const auto insertionPoint = absl::c_upper_bound(events, delay, MidiEventDelayComparator {});
+    const auto insertionPoint = absl::c_lower_bound(events, delay, MidiEventDelayComparator {});
```

Comment:
Considering there's a test below to check if the `delay` in the new event is the same as the `insertionPoint` one, I think I meant to use `lower_bound` here. If we have two events with the same delay in a block, the last received one is supposed to overwrite the previous one.